File: Java/Easy/src/FirstIndex.java
public class FirstIndex {

    public int firstIndex(String haystack, String needle) {
        return haystack.indexOf(needle);
    }

    public static void main(String[] args) {
        FirstIndex fi = new FirstIndex();
        // Example 1
        String haystack1 = "sadbutsad";
        String needle1 = "but";
        System.out.println(fi.firstIndex(haystack1, needle1));

        // Example 2
        String haystack2 = "leetcode";
        String needle2 = "leeto";
        System.out.println(fi.firstIndex(haystack2, needle2));
    }
}

File: JS/Easy/firstIndex.js
let firstIndex = function(haystack, needle) {
    return haystack.indexOf(needle);
}

let main = function() {
    // Example 1
    let haystack1 = 'sadbutsad';
    let needle1 = 'but';
    console.log(firstIndex(haystack1, needle1));

    // Example 2
    let haystack2 = 'leetcode';
    let needle2 = 'leeto';
    console.log(firstIndex(haystack2, needle2));
}

main();